Content/Items/Gear/Weapons/WarShields/StarlightBulwark.cs

```diff
@@ -0,0 +1,257 @@
+﻿using PathOfTerraria.Core.Systems;
+using PathOfTerraria.Core.Systems.Affixes;
+using PathOfTerraria.Core.Systems.Affixes.ItemTypes;
+using PathOfTerraria.Core.Systems.ModPlayers;
+using ReLogic.Content;
+using System.Collections.Generic;
+using Terraria.DataStructures;
+using Terraria.ID;
+
+namespace PathOfTerraria.Content.Items.Gear.Weapons.WarShields;
+
+internal class StarlightBulwark : LeadBattleBulwark
+{
+	public static Asset<Texture2D> GlowTexture;
+
+	public override bool IsUnique => true;
+	public override ShieldData Data => new(15, 160, 14f, DustID.YellowStarDust);
+
+	private int _projTimer = 0;
+	private int _lastProj = -1;
+	private int _lastAltClick = 0;
+	private bool _specialAlt = false;
+
+	public override void SetStaticDefaults()
+	{
+		GlowTexture = ModContent.Request<Texture2D>(Texture + "Shine");
+	}
+
+	public override List<ItemAffix> GenerateAffixes()
+	{
+		var addedDamageAffix = (ItemAffix)Affix.CreateAffix<IncreasedDamageAffix>(-1, 15, 25);
+		var attackSpeedAffix = (ItemAffix)Affix.CreateAffix<ManaAffix>(-1, 15, 20);
+		var noFallDamageAffix = (ItemAffix)Affix.CreateAffix<NoFallDamageAffix>(-1, 1f, 4f);
+		return [addedDamageAffix, attackSpeedAffix, noFallDamageAffix];
```

Comment:
Is there any issues with only having 3 affixes?

Reply:
Hm? Why would there be?

Reply:
I remember seeing some logic when I was working on data driven affixes that it would fill in affixes that were missing from the instantiated count 4 array.
If it doesn't fill in some random affix when it's spawned/dropped it should be fine.

Reply:
That sounds like a structural issue we should fix, functionality like that doesn't make any sense. I didn't look too hard but I don't believe it had 4 affixes when I tested, just 3.